apps/src/clientApi.js

```diff
@@ -442,9 +433,44 @@ class FilesApi extends CollectionsApi {
     );
   }
 }
+
+class StarterAssetsApi {
+  getStarterAssets(levelChannelId, onSuccess, onFailure) {
+    return ajaxInternal(
+      'GET',
+      this.withLevelChannelId(levelChannelId).basePath(''),
```

Comment:
within js code, `levelStarterAssetsId` could be a good alternative (to eliminate `ChannelId`).